src/components/Link.js

```diff
@@ -36,13 +36,13 @@ class CSVLink extends React.Component {
   /**
    * In IE11 this method will trigger the file download
    */
-  handleLegacy(event, data, headers, separator, filename) {
+  handleLegacy(evt, data, headers, separator, filename, enclosingCharacter) {
```

Comment:
Don't know why this was changed. Will break line 43?

Reply:
My apologies, I merged this hastily because it had been sitting around for some time. I'll resolve these problems and merge into master.

Reply:
@mriccid no worries. My fault for not getting to this PR sooner.